modules/fedimint-wallet-server/src/lib.rs

```diff
@@ -1206,6 +1184,84 @@ impl Wallet {
             secp: &self.secp,
         }
     }
+
+    fn spawn_broadcast_pending_task(
+        task_group: &mut TaskGroup,
+        bitcoind: &DynBitcoindRpc,
+        db: Database,
+    ) {
+        task_group.spawn("broadcast pending", {
+            let bitcoind = bitcoind.clone();
+            let db = db.clone();
+            |handle| async move {
+                run_broadcast_pending_tx(db, bitcoind, &handle).await;
+            }
+        });
+    }
+
+    fn spawn_bitcoin_update_task(
+        task_group: &mut TaskGroup,
+        bitcoind: &DynBitcoindRpc,
+    ) -> (Arc<AtomicU32>, Arc<RwLock<Option<Feerate>>>) {
+        let block_count_bg = Arc::new(AtomicU32::new(0));
+        let fee_rate_bg = Arc::new(RwLock::new(None));
```

Comment:
Why not initialize with default from config instead of making it an `Option` if we `unwrap_or` anyway later?